sources/dataset.py

```diff
@@ -32,15 +32,12 @@
 
 import utils
 
-
 class Dataset(object):
-
-    def __init__(self, image_size, glossary_filename):
-        """ Class constructor
-        """
+    """
```

Comment:
Maybe you can add some content to the docstring of the class.

Reply:
Done